build/teamcity/cockroach/ci/tests/unit_tests_impl.sh

```diff
@@ -3,6 +3,14 @@
 set -xeuo pipefail
 
 bazel build //pkg/cmd/bazci --config=ci
+
+extra_params=""
```

Comment:
Nit: Can you use capital letters like `EXTRA_PARAMS`? It's just more consistent with the rest of our CI scripts.

Reply:
Sure thing, fixing.